nip47/permissions/permissions.go

```diff
@@ -79,6 +79,11 @@ func (svc *permissionsService) GetPermittedMethods(app *db.App, lnClient lnclien
 	// only return methods supported by the lnClient
 	lnClientSupportedMethods := lnClient.GetSupportedNIP47Methods()
 	requestMethods = utils.Filter(requestMethods, func(requestMethod string) bool {
+		// TODO: better way to exclude methods unrelated to the lnclient
+		if requestMethod == models.CREATE_CONNECTION_METHOD {
+			return true
+		}
```

Comment:
We can remove this as currently this is not hit at all since `GetSupportedNIP47Methods` doesn't return `create_connection`

Reply:
If we can have a function like `GetServiceSupportedNIP47Methods` then it could be used both here and in `api.GetWalletCapabilities` like mentioned above

Reply:
Thanks for checking. Currently we don't return it in `api.GetWalletCapabilities`. For now I would keep it as-is. I'm not sure there is any point returning service capabilities because it's constant, no matter what wallet backend